airflow/executors/all_executors.py

```diff
@@ -0,0 +1,88 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""All executors."""
+from typing import Optional
+
+from airflow import AirflowException, LoggingMixin, conf
+from airflow.executors.base_executor import BaseExecutor
+
+
+class AllExecutors:
```

Comment:
```suggestion
class Loader:
```

(and rename to loader.py)

Reply:
Sure. I think it's a better name. I will actually name it ExecutorLoader instead as Loader is rather generic.